### DISCUSSION TITLE
Looking for a way to group input fields in columns / make forms more attractive

### DISCUSSION THREAD

**Discussion by anxgstadler** [Laravel-Backpack/community-forum#1192] [Q&A (Help)]
Hi, everyone!

I'm new to Backpack and as I see it, I can add classes to my inputs so they are less wide, thus getting 2 or more inputs in one row next to each other.

What I'm looking for is to have one set of inputs in one column and another set of inputs in a second column. An example might be a big main column with article content and a smaller secondary column with meta infos.
(Some kind of compromise would be to put those meta infos in a separate tab, but this would hide them from the user while having more than enough space on the screen.)

Does backback already provide this functionality? And if not, what would be the best way to approach this?

Generally speaking, the documentation lacks information on my possibilities to make forms more visually attractive for users. If e.g. I want to write this 2-column-structure as my own view, how would I then get the inputs form the CRUD controller in there?
A small tutorial for making my own form view would really help here! :-)
(And if I overlooked it, I do apologize. Please just point me in the right direction...)

**Comment by karandatwani92** [comment]
Hey @anxgstadler 

I understand you are looking to set up a form like this:

![image](https://github.com/user-attachments/assets/391b59a6-3d40-4b1d-86c1-3342fe18c0a7)

I discovered this feature in a package, but its last update was 7 years ago. That means it's dead. 
https://github.com/novius/laravel-backpack-crud-extended

Unfortunately, this feature is currently not available in Backpack. For now, You can only split your create/edit inputs into [multiple tabs](https://backpackforlaravel.com/docs/6.x/crud-fields#split-fields-into-tabs)


You can ask/vote for this feature request [here](https://github.com/Laravel-Backpack/community-forum/discussions/751). It may come in the next version if votes are high.

**Reply by karandatwani92** [reply]
In case you create this feature yourself, you can choose to submit a [PR](https://github.com/Laravel-Backpack/CRUD/pulls) and share it with the Backpack community.

**Comment by anxgstadler** [comment]
I really wonder why this section is called "discussions" when you close a thread directly after your very first answer ;-)
(It comes across a bit rude, as the thread then completely vanishes from the discussions page with its default "is:open" filter...)

While closing the thread, you ignored the most important part of my initial post: If the feature is not available currently, what would be the best way to approach this? I mean, I could shell out the structure in a custom view and tell the CRUD controller to use this view for create and edit, but how would I then render the fields there? Is there somehting like `<x-backpack-field name="my-field">` ?

Thanks! :-)

(P.S. Please forgive me, but I don't have too much hope regarding voting for the feature on the other page, since the other topic I was asking for, managing linked resources on the show page of an entity, has been up-voted there to number 2 a year ago, and it's still not there ;-) )

**Comment by karandatwani92** [comment]
Oh! I'm sorry to close it very soon.  Thanks for correcting me.
Yes... I'll try to explore the possible ways and come up with a conclusion.

**Reply by anxgstadler** [reply]
Thanks!
(And I'm sorry if I, myself, was coming across as rude. I'm evaluating if I want to shell out 400 bucks on this project, that's why I have to get most of my questions cleared. I hope you understand.)

I had a look at the demo repository: The "monsters" section is really impressive, given the number of all the features. But to make this mega-form attractive for users would really be a challenge. How would one approach this?

**Comment by JohnHamagami** [comment]
I think this is a super important feature. tabbing between field sets means that someone may miss an important piece of information that could have easily been displayed in a more compact format. 

Since you already have tabs, it would be fairly easy (for you - LOL) to use the same logic to create a lay out with panels likes this.. then one would define the fields inside each panel.. or add more if need. so panel_left_1 panel_left_2 > panel_right_1 panel pane_right_2 and you can define the left or right with like 2/3 etc.
